rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/domain/service/LRAService.java

```diff
@@ -215,6 +220,7 @@ public void remove(URI lraId) {
 
         recoveringLRAs.remove(lraId);
 
+        LRALogger.logger.infof("LRAService: removed LRA from cache: ", lraId);
```

Comment:
@mmusgrov this one is missing the "%s" in the message

Reply:
Well spotted, thanks. Note that I removed all of the extra debug in favour of debugging just the enlistParticipant method [1].

[1] https://github.com/jbosstm/narayana/blob/master/rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LongRunningAction.java#L717